static/css/responsive.css

```diff
@@ -0,0 +1,263 @@
+#container,
+#footer,
+#footer_contents,
+#main,
+#no-flickr,
+#events_content,
+#main-contents,
+#sides,
+#wiki-container,
+#favorites,
+#main-side,
+#likebox,
+#opportunities_content,
+#small-side,
+#blog_contents,
+#blog_main,
+#all_about_us,
+#klab-map,
+#about_detail,
+input.form-control,
+textarea.form-control,
+select.form-control,
+.person-words,
+.about_details,
+.questions,
+.paginate,
+.full_post,
+.row,
+.post_main,
+.event_small,
+.event_main,
+.event_heading,
+.description,
+.pull-left,
+.pull-right,
+.event_details,
+.full-events,
+.opportunity_small,
+.search_form input,
+.full_opportunities,
+.opportunity_main,
+.contact input,
+.contact textarea,
+.col-md-4 img,
+.cancel,
+.opportunity_details,
+iframe{
+	width:100%;
```

Comment:
No if change the above code to div, iframe, input, textarea {width:100%} , we'll still get left to right scrolling.